src/pages/dashboard/settings/Plans.vue

```diff
@@ -155,6 +160,75 @@ const hightLights = computed<Stat[]>(() => ([
     value: currentPlanSuggest.value?.name,
   },
 ]))
+
+const supabase = useSupabase()
+
+async function getUsage() {
+  const apps = await supabase.from('apps')
+    .select('app_id')
+    .eq('user_id', main!.user!.id!)
+
+  if (!apps.data)
+    return
+
+  const usage = await supabase.from('app_usage')
+    .select('mau, storage, bandwidth')
+    .gte('created_at', main.cycleInfo?.subscription_anchor_start)
+    .lte('created_at', main.cycleInfo?.subscription_anchor_end)
+    .in('app_id', apps.data.map(app => app.app_id))
+
+  const plan = plans.value.find(p => p.name === 'Pay as you go')!
```

Comment:
Will we always have a value here?

Reply:
Yes, this table is only for pay as you go users

Reply:
Got it, thanks for explaining!